src/modules/legacy.js

```diff
@@ -347,6 +353,11 @@ const onTwitchMessage = async (controller, channel, user, message, tags) => {
 
 	// logger.log("Valid Command",user,userCommand, tags.userInfo);
 
+	if (config.useNewCommandSystem.has(userCommand) && userCommand !== 'uselegacy') {
+		controller.commandManager.handleTwitchMessage(channel, user, message, tags);

```

Comment:
I'm not sure I completely follow how this is working? The command manager also binds to incoming Twitch messages? `controller.connections.twitch.onMessage(this.handleTwitchMessage.bind(this));`